frame/metadata/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "frame-metadata"
-version = "2.0.0"
+version = "10.0.0"
```

Comment:
?!

Reply:
I think it would be nice to let the crate version follow the actual metadata version. We will have new metadata versions and each will require a major bump.